stopify-continuations/src/types.ts

```diff
@@ -3,6 +3,7 @@ export type HandleNew = 'direct' | 'wrapper'
 
 export interface CompilerOpts {
   compileFunction?: boolean,
+  getters: boolean,
```

Comment:
how about `getters?`. This will avoid breaking the website. I think some sensible defaults will also help.

Reply:
to be clear, some "sensible defaults" will help a user of Stopify. The number of options is otherwise overwhelming.

Reply:
I think options for language features like `hof`, `jsArgs`, and `getters` should always be explicitly specified by the interface user.

I can change the website to pass the value for getters.

Reply:
OK. I disagree. As I said, too many options is overwhelming.